Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsController.cs

```diff
@@ -577,5 +609,60 @@ private void AddProblemGroupsToContest(Contest contest, int problemGroupsCount)
                 this.participantsData.InvalidateByContestAndIsOfficial(contest.Id, isOfficial: false);
             }
         }
+
+        private string GetMessageForChangedParticipantsEndTimeByTimeInterval(
+            int timeInMinutes,
+            string contestName,
+            ICollection<string> notUpdatedParticipantUsernames)
+        {
+            var minutesForDisplay = timeInMinutes.ToString();
+
+            var sb = new StringBuilder();
```

Comment:
Using StringBuilder implies a lot of string concatenation, and this is not such a case. Simple string is better IMO.